pkgs/top-level/python-aliases.nix

```diff
@@ -104,6 +104,7 @@ mapAliases ({
   face_recognition_models = face-recognition-models; # added 2022-10-15
   fake_factory = throw "fake_factory has been removed because it is unused and deprecated by upstream since 2016."; # added 2022-05-30
   faulthandler = throw "faulthandler is built into ${python.executable}"; # added 2021-07-12
+  inherit fetchPypi;
```

Comment:
I would actually prefer a deprecation notice here. Maybe that's too unfriendly?

Reply:
as in `throw` or `warn`? I think adding a warning here makes sense

Reply:
As in `throw`, I mean.

Reply:
I think we might want to wait a little longer before converting this to `throw`